From: Nikias Bassen <nikias@gmx.li>
Date: Fri, 7 Aug 2020 00:50:46 +0200
Subject: mobilebackup2: Set DeviceLink version to 400 to support iOS 14b4+

---
 src/mobilebackup2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mobilebackup2.c b/src/mobilebackup2.c
index 56b7058..6c6556d 100644
--- a/src/mobilebackup2.c
+++ b/src/mobilebackup2.c
@@ -30,7 +30,7 @@
 #include "device_link_service.h"
 #include "common/debug.h"
 
-#define MBACKUP2_VERSION_INT1 300
+#define MBACKUP2_VERSION_INT1 400
 #define MBACKUP2_VERSION_INT2 0
 
 #define IS_FLAG_SET(x, y) ((x & y) == y)
